packages/gatsby/cache-dir/static-entry.js

```diff
@@ -103,315 +103,325 @@ export default ({
   reversedStyles,
   reversedScripts,
 }) => {
-  let bodyHtml = ``
-  let headComponents = [
-    <meta
-      name="generator"
-      content={`Gatsby ${gatsbyVersion}`}
-      key={`generator-${gatsbyVersion}`}
-    />,
-  ]
-  let htmlAttributes = {}
-  let bodyAttributes = {}
-  let preBodyComponents = []
-  let postBodyComponents = []
-  let bodyProps = {}
-
-  const replaceBodyHTMLString = body => {
-    bodyHtml = body
-  }
-
-  const setHeadComponents = components => {
-    headComponents = headComponents.concat(sanitizeComponents(components))
-  }
-
-  const setHtmlAttributes = attributes => {
-    htmlAttributes = merge(htmlAttributes, attributes)
-  }
-
-  const setBodyAttributes = attributes => {
-    bodyAttributes = merge(bodyAttributes, attributes)
-  }
-
-  const setPreBodyComponents = components => {
-    preBodyComponents = preBodyComponents.concat(sanitizeComponents(components))
-  }
+  // for this to work we need this function to be sync or at least ensure there is single execution of it at a time
+  global.unsafeBuiltinUsage = []
```

Comment:
This hurts my feelings, but I essentially need "context" that "unconnected" arbitrary code can access and push to.